kmath-geometry/src/commonMain/kotlin/scientifik/kmath/geometry/RealEuclideanVectorSpace.kt

```diff
@@ -0,0 +1,75 @@
+package scientifik.kmath.geometry
+
+import scientifik.kmath.dimensions.*
+import scientifik.kmath.operations.ExtendedField
+import scientifik.kmath.operations.RealField
+import kotlin.math.sqrt
+
+data class RealVector<D : Dimension>(private val array: DoubleArray) : Vector<Double, D> {
+    override val size: Int = array.size
```

Comment:
Can be replaced with getter since arrays usually store their lengths by themselves. (at least on JVM)

Reply:
replaced with getter